climada/engine/impact.py

```diff
@@ -390,6 +388,44 @@ def impact_per_year(self, all_years=True, year_range=None):
             year_set[year] = sum(self.at_event[orig_year == year])
         return year_set
 
+    def impact_at_reg(self, agg_regions=None):
+        """Aggregate impact on given aggregation regions. This method works 
+        only if Impact.imp_mat was stored during the impact calculation.
+
+        Parameters
+        ----------
+        agg_regions : np.array, list, pd.Series
+            It has the same lenght as the number of coordinates points in exposure 
+            and reports what macro-regions these points belong to. If no aggregation
+            regions are passed, the method aggregates impact at the admin_0 level.
+            Default is None.
+
+        Returns
+        -------
+        pd.DataFrame
+        """
```

Comment:
Maybe add a usage example? The input parameter `agg_regions` is a bit unintuitive.

Reply:
mhh, I am a bit reluctant to add a real example because this is not done in CLIMADA not event for much more complex functions. I added a thorough description of what these `agg_regions` are in the docstring.

Reply:
Great! Thank you